src/components/store/OrderSummary/index.tsx

```diff
@@ -26,22 +34,44 @@ const isOrderActionable = ({ status, pickupEvent }: PublicOrderWithItems): boole
   return true;
 };
 
-const OrderSummary = ({ order }: OrderSummaryProps) => {
+const OrderSummary = ({
+  order,
+  orderStatus,
+  futurePickupEvents,
+  pickupEvent,
+  reschedulePickupEvent,
+  cancelOrder,
+}: OrderSummaryProps) => {
   const items = getOrderItemQuantities(order.items);
 
   const totalCostWithoutDiscount = items.reduce((cost, item) => {
     return cost + item.quantity * item.option.price;
   }, 0);
 
-  const actionable = isOrderActionable(order);
+  const [modalOpen, setModalOpen] = useState<boolean>(false);
+  const [cancelModalOpen, setCancelModalOpen] = useState<boolean>(false);
+  const actionable = isOrderActionable({ ...order, status: orderStatus, pickupEvent });
 
   return (
     <div className={styles.container}>
+      <PickupEventPreviewModal
+        open={modalOpen}
+        onClose={() => setModalOpen(false)}
+        pickupEvent={pickupEvent}
+        reschedulePickupEvent={reschedulePickupEvent}
+        futurePickupEvents={futurePickupEvents}
+        reschedulable
```

Comment:
it's still reschedulable for cancelled orders. is this intended?

Reply:
fixed!